src/main/java/com/redblueflame/herbocraft/HerboCraft.java

```diff
@@ -79,7 +79,7 @@ public void onInitialize() {
 
                     if (!cropBlock.isMature(state)) {
                         cropBlock.randomTick(state, (ServerWorld) world, pos, world.random);
-                        stack.damage(10, world.random, player);
+                        player.inventory.getMainHandStack().damage(5, world.random, player);
```

Comment:
Can you share in the packet the position in inventory of the watering can, and use it in here ?